<ui version="4.0" >
 <class>mdlgNormalize</class>
 <widget class="QDialog" name="mdlgNormalize" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Normalizing Data</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>520</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Normalize Data:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>500</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>105</y>
     <width>351</width>
     <height>177</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QCheckBox" name="mchkBoxSp" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>152</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Z : Normalized Sp = ln(Sp)/10</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text" >
     <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Peptide Length&lt;/span> is calculated by first trimming the leading &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">and trailing residues and then removing all characters that &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">do not correspond to an amino acid.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Ex: &lt;span style=" font-weight:600;">R.IYEVSQ*LAK.A &lt;/span>will become &lt;span style=" font-weight:600;">IYEVSQLAK &lt;/span>and &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">the length will be &lt;span style=" font-weight:600;">9.&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;">&lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="textFormat" >
     <enum>Qt::RichText</enum>
    </property>
    <property name="margin" >
     <number>3</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxDelCn2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>128</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Y : Normalized DelCn2 = √(DelCn2)</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxXCorr" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>X : Normalized XCorr = ln(XCorr)/ln(Peptide Length)</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>320</y>
     <width>351</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QCheckBox" name="mchkBoxHyperS" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>X : Normalized Hyperscore = ln(Hyperscore)</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxDeltaCn2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Y : Normalized DeltaCn2 = DeltaCn2 (no change)</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="mrBtnSequest" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Normalize Sequest Scores</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnXTandem" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>293</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Normalize X!Tandem Scores</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnSkip" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Do Not Alter Data</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtnSpectrumMill" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>400</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Normalize Spectrum Mill Scores</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>426</y>
     <width>351</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QCheckBox" name="mchkBoxSMScore" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>X : Normalized Score = √(Score)</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxSMDelta" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Y : Normalized ΔFwdRev = √|ΔFwdRev|</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdlgNormalize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
